### PR TITLE
Add active series to all user stats page

### LINKED ISSUE #4917: Add active series  metric to ingester stats page
**Is your feature request related to a problem? Please describe.**
Now, the cortex ingester stats page  has some series yet, is it possible to add the `cortex_ingester_active_series` metric?

image

### PR DESCRIPTION
**Pull request by songjiayang**
Signed-off-by: songjiayang <songjiayang1@gmail.com>

**What this PR does**:

Add active_series to all_user_stats page.

**Which issue(s) this PR fixes**:
Fixes #4917

**Checklist**
- [x] Tests updated
- [ ] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Comment by alanprot** [comment]
NIce!! :D

Can you fix the lint?

**Comment by songjiayang** [comment]
Thank @alanprot,  I have changed `protoc-gen-gogoslick` version from v1.3.2 to v1.3.0 on my local, code generated looks fine. review again please.

File: pkg/distributor/distributor.go [hunk 1]
```diff
@@ -1189,10 +1189,13 @@ func (d *Distributor) UserStats(ctx context.Context) (*UserStats, error) {
...
 		totalStats.RuleIngestionRate += r.RuleIngestionRate
 		totalStats.NumSeries += r.NumSeries
+		totalStats.ActiveSeries += r.ActiveSeries
 	}
 
-	totalStats.IngestionRate /= float64(d.ingestersRing.ReplicationFactor())
-	totalStats.NumSeries /= uint64(d.ingestersRing.ReplicationFactor())
+	factor := d.ingestersRing.ReplicationFactor()
+	totalStats.IngestionRate /= float64(factor)
+	totalStats.NumSeries /= uint64(factor)
```
Comment (yeya24) [inline pkg/distributor/distributor.go:1197]: So previously number of series is not divided by the factor? Do we want to change this in this pr?

Comment (songjiayang) [reply pkg/distributor/distributor.go:1197]: previously number of series is divided here, I just change `d.ingestersRing.ReplicationFactor()` to a uniform variable

Comment (yeya24) [reply pkg/distributor/distributor.go:1197]: Make sense.

**Review by yeya24 — APPROVED** [review]
Thanks for the contribution! This looks great.

**Review by alvinlin123 — APPROVED** [review]

**Pull request merged**